### PR TITLE
Fix type-safety around redshift connection profiles. The code tried to pass a Long to the redshift API with no type errors being thrown until inside that API.

### PR DESCRIPTION
**Pull request by BenBirt**

### CODE REVIEW

File: api/dbadapters/index.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export function create(profile: protos.IProfile, warehouseType?: string): DbAdap
     return new registry[warehouseType](profile);
   }
   if (!!profile.bigquery) {
-    return new registry["bigquery"](profile);
+    throw new Error(profile.toString());
```
Comment (lewish) [inline api/dbadapters/index.ts]: This doesn't look right...

File: protos/profiles.proto [hunk 2]
```diff
@@ -19,10 +19,16 @@ message Profile {
 
 message JDBC {
   string host = 2;
-  int64 port = 3;
```
Comment (lewish) [inline protos/profiles.proto]: My understanding was these are backwards compatible, so you could just change this and save quite a bit of work here? https://developers.google.com/protocol-buffers/docs/proto#updating

No one has set a non 32-bit port number so we don't need to worry about that :)

File: api/dbadapters/index.ts (hunk 1 above)
Comment (BenBirt) [reply api/dbadapters/index.ts]: Oops, sorry, debugging stuff, didn't intend it to make it here.

File: protos/profiles.proto (hunk 2 above)
Comment (BenBirt) [reply protos/profiles.proto]: Interesting, I didn't know this was safely change-compatible, good to know.

**Pull request merged**